orttraining/orttraining/python/training/execution_agent.py

```diff
@@ -9,9 +9,72 @@
 from onnxruntime.capi._pybind_state import TrainingAgent as C_TrainingAgent
 
 
+class ExecutionAgentOutput(object):
+    def __init__(self, ortvalues, run_id=None):
+        self.ortvalues = ortvalues
+        self.run_id = run_id
+
+
+class InferenceAgent(object):
+    """
+    This is the main class used to run a ORTModule model inferencing.
```

Comment:
Typo nit: "a ORTModule" -> "an ORTModule". (Also below in the TrainingAgent docstring.)